src/network/netdev/macsec.c

```diff
@@ -20,6 +20,12 @@
 #include "string-util.h"
 #include "unaligned.h"
 
+#define SECURITY_ASSOCIATION_NULL               \
```

Comment:
we usually would put a `(SecurityAssociation)` before the `{`...

doesn#t really matter too much, but is nice, in case people write:

```c
SecurityAssociation sa;
sa = SECURITY_ASSOCIATION_NULL;
```

because right now this will fail, but if you add the `(SecurityAssociation)` prefix it will work.

Now, strictly speaking, this relies on a GNU extension, since in C99 this is not actually allowed:

```c
SecurityAssociation sa = (SecurityAssociation) { … };
```

but GNU allows that.

(also I think in newer C standards they explicitly allowed that after all, or i might be hallucinating).

nothing of this matters really, just wanted to type a lot here ;-)